zip-0227.rst

```diff
@@ -1,8 +1,359 @@
+========
+
+ZIP 226 [#zip-0226]_ and ZIP 227 [#zip-0227]_ propose in conjunction the Zcash Shielded Assets (ZSA) protocol, which is an extension of the Orchard protocol that enables the creation, transfer and burn of custom assets on the Zcash chain. The creation of such assets is defined in ZIP 227 [#zip-0227]_. The transfer and burn of such assets is defined in ZIP 226 [#zip-0226]_. This ZIP must only be implemented in conjuction with ZIP 226 [#zip-0226]_, as the issuance mechanism is only valid for the ZSA transfer protocol, as it produces notes that can only be transferred under ZSA.
+
+Motivation
+==========
+
+The Orchard protocol deployed in NU5 does not support custom assets. Enabling multi-asset support on the Zcash chain will open the door for a host of applications and enhance the ecosystem with application developers and asset custody institutions for issuance and
+bridging purposes.
+
+Use Cases
+=========
+
+The design presented in this ZIP enables issuance of shielded assets in various modes:
+
+- The issuer knows in advance the receivers of the issued asset.
+- The asset can be of non-fungible type, where each asset type can be made part of a single “series”.
+- The supply of the asset can be limited in advance or not.
+- The asset can be a wrapped version of an asset issued by another chain (as long as there is a bridge that supports transfer of that asset between chains).
+
+See the Concrete Applications section for more details.
+
+Requirements
+============
+
+- Any user of the Zcash block chain can issue custom assets on chain
+- The issuance mechanism should enable tracking publicly the supply of the assets on the Zcash block chain
+- Issuing or changing the attributes of a specific asset should require cryptographic authorization.
+- The asset identification should be unique (among all shielded pools) and different issuer public keys should not be able to generate the same asset identifier.
+
+
+Specification
+=============
+
+As explained, the issuance protocol allows for a single issuance action to be sent to many receivers, by generating as many output notes as distinct recipients. Furthermore, every bundle can contain many issuance actions.
+
+Overview 
+--------
+
+This specification enables only *transparent* issuance, since as a first step, transparency is important to properly test the applications that will be most used in the Zcash ecosystem, while enabling tracking of supply of assets. Any user of the blockchain can issue new asset types, yet only a single issuer (a set of keys) can issue that specific asset type, as the asset identifiers are tied to the issuer keys.
+
+The protocol was designed with three main properties in mind:
+- at the time of issuance, the issuer can already allocate all the tokens to the corresponding owners by creating the corresponding (shielded) output notes to the respective addresses. As is implied, the issuance mechanism is itself transparent, but the allocation is totally private
```

Comment:
Down in the `IssueAction` description, we see a lot of the detail of the notes; what does 'allocation is totally private' mean?

Reply:
Specifically, the notes are not encrypted, so we will see who gets allocated how much of each issued asset.

Reply:
We fixed this to not be ambiguous.